images/fcos-bastion-image/root/usr/share/bastion_services/dhcp/tftpboot/grub.cfg

```diff
@@ -0,0 +1,11 @@
+set timeout=30
```

Comment:
This is the current grub.cfg

```suggestion
set timeout=30
set default=0
insmod efi_gop
insmod efi_uga
load_video

if [ "${net_efinet0_extensionspath}${net_efinet1_extensionspath}${net_efinet2_extensionspath}${net_efinet3_extensionspath}${net_efinet4_extensionspath}" = "x86_64" ]; then
 menuentry 'Wipe disk (fcos-x86_64)' {
    set gfx_payload=keep
    insmod gzio
    linux  /fcos-x86_64/fedora-coreos--live-kernel-x86_64 debug console=tty0 console=ttyS0,115200 coreos.live.rootfs_url=http://192.168.90.1/fcos-x86_64/fedora-coreos-live-rootfs.x86_64.img ignition.firstboot ignition.platform.id=metal ignition.config.url=http://192.168.90.1/wipe_disks.ign
    initrd /fcos-x86_64/fedora-coreos-live-initramfs.x86_64.img
};
menuentry 'Fedora CoreOS Shell (x86_64)' {
    set gfx_payload=keep
    insmod gzio
    linux  /fcos-x86_64/fedora-coreos-live-kernel-x86_64 console=tty0 console=ttyS0,115200 coreos.live.rootfs_url=http://192.168.90.1/fcos-x86_64/fedora-coreos-live-rootfs.x86_64.img ignition.firstboot ignition.platform.id=metal ignition.config.url=http://192.168.90.1/shell.ign
    initrd /fcos-x86_64/fedora-coreos-live-initramfs.x86_64.img
};
elif [ "${net_efinet0_extensionspath}${net_efinet1_extensionspath}${net_efinet2_extensionspath}${net_efinet3_extensionspath}${net_efinet4_extensionspath}" = "aarch64" ]; then

menuentry 'Wipe disk (fcos-aarch64)' {
    set gfx_payload=keep
    insmod gzio
    linux  /fcos-aarch64/fedora-coreos-live-kernel-aarch64 debug console=tty0 console=ttyS0,115200 coreos.live.rootfs_url=http://192.168.90.1/fcos-aarch64/fedora-coreos-live-rootfs.aarch64.img ignition.firstboot ignition.platform.id=metal ignition.config.url=http://192.168.90.1/wipe_disks.ign
    initrd /fcos-aarch64/fedora-coreos-live-initramfs.aarch64.img
};
menuentry 'Fedora CoreOS Shell (aarch64)' {
    set gfx_payload=keep
    insmod gzio
    linux  /fcos-aarch64/fedora-coreos-live-kernel-aarch64 debug console=tty0 console=ttyS0,115200 coreos.live.rootfs_url=http://192.168.90.1/fcos-aarch64/fedora-coreos-live-rootfs.aarch64.img ignition.firstboot ignition.platform.id=metal ignition.config.url=http://192.168.90.1/shell.ign
    initrd /fcos-aarch64/fedora-coreos-live-initramfs.aarch64.img
};

fi

menuentry "Reload GRUB" {
    configfile /grub.cfg
}

```


`shell.ign` and `wipe_disks.ign` might be generated similarly to the idea @zniu1011 had for #13.

You can make templates in /usr/share/bastion_services/ignitions for both of them without the authorized keys and the passwordHash.

Consider that two files will be provided under /var/opt/ignitions/:
- `/var/opt/ignitions/authorized_keys`
- `/var/opt/ignitions/password_hashes`

Then, 
1. add a script that generates `/opt/html/wipe_disks.ign` and `/opt/html/shell.ign`.
2. Create a dnsmasq-init.service as in https://github.com/openshift-qe/baremetal-qe-infra/pull/14#discussion_r1302526332 (also see https://docs.fedoraproject.org/en-US/packaging-guidelines/Initial_Service_Setup/) as a dependency of dnsmasq.service


The dnsmasq-init.service should:
1. Execute the script to generate the ignitions (you could use butane)
2. [optional] download the latest fcos
3. Extract `grubaa64.efi` and `grubx64.efi` from RPMs

To extract from the RPMs, you should run the following commands (make a script) within a CentOS stream 9 containter (quay.io/centos/centos:stream9):
```
dnf install -y cpio 
cd /tmp
mkdir -p {aarch64,x86_64,output}
curl -L -O $(dnf repoquery --location grub2-efi-aa64 --forcearch aarch64 | tail -n 1)
curl -L -O $(dnf repoquery --location grub2-efi-x64 --forcearch x86_64 | tail -n 1)
mkdir -p {aarch64,x86_64}

cd /tmp/aarch64
rpm2cpio /tmp/grub2-efi-aa64* | cpio -idmv
mv boot/efi/EFI/centos/*.efi /tmp/output/

cd /tmp/x86_64
rpm2cpio /tmp/grub2-efi-x64* | cpio -idmv
mv boot/efi/EFI/centos/*.efi /tmp/output/
```

```
openshift-qe-bastion tftpboot # cat /opt/html/wipe_disks.ign
{
  "ignition": {
    "version": "3.3.0"
  },
  "passwd": {
    "users": [
      {
        "name": "core",
        "sshAuthorizedKeys": [
          "*** OMITTED ***"
        ]
      }
    ]
  },
  "systemd": {
    "units": [
      {
        "contents": "[Unit]\nDescription=Wipe the disks\nOnFailure=emergency.target\nOnFailureJobMode=replace-irreversibly\nAfter=network-online.target\nWants=network-online.target\n[Service]\nType=oneshot\nExecStart=bash -c 'for i in $(lsblk -I8 -nd --output name); do dd if=/dev/zero of=/dev/$i bs=1M count=16; done'\nExecStart=bash -c '/usr/sbin/efibootmgr -v | sed -nE \"s|^Boot([0-9A-Fa-f]{4}).*HD\\(.*$|\\1|gp\" | xargs -I% /usr/sbin/efibootmgr --delete-bootnum --bootnum %'\nExecStart=/usr/bin/systemctl --no-block poweroff\nStandardOutput=kmsg+console\nStandardError=kmsg+console\n[Install]\nRequiredBy=default.target\n",
        "enabled": true,
        "name": "wipe-disks.service"
      }
    ]
  }
}
```

```
{
  "ignition": {
    "version": "3.3.0"
  },
  "passwd": {
    "users": [
      {
        "name": "core",
        "passwordHash": "*** OMITTED ***",
        "sshAuthorizedKeys": [
          "*** OMITTED ***"
        ]
      }
    ]
  }
}
```

Reply:
That's not the same grub.cfg (it includes fcos versions)

Reply:
fixed